Resources/views/Collector/db.html.twig

```diff
@@ -359,10 +359,15 @@
                     </tr>
                     </thead>
                     <tbody>
-                    {% for class in classes %}
+                    {% for class in classes|keys %}
```

Comment:
since class is part of array, I think it would be better to change `class` instances below with `class.class` instead of iterating over the keys and then getting that same array through `classes[class]`